### PR TITLE
Helper function for loading a csv file.  People might find this useful for hooks

### PR DESCRIPTION
**Pull request by silasary**

### CODE REVIEW

File: src/Helpers.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
+import csv
 import os
 import pkgutil
 import json
 
-from BaseClasses import MultiWorld, Item, Location
```
Comment (FuzzyGamesOn) [inline src/Helpers.py:5]: ManualLocation comes from Location and is still used below, so we need to add this back.

**Review by FuzzyGamesOn — CHANGES_REQUESTED** [review]
Just the one fix, everything else looks good.

File: src/Helpers.py (hunk 1 above)
Comment (silasary) [reply src/Helpers.py:5]: It's not used at all in this file

Comment (FuzzyGamesOn) [reply src/Helpers.py:5]: Oh, you're right. Misread, carry on :smiley:

**Review by nicopop — APPROVED** [review]
Looks good.

**Review by FuzzyGamesOn — APPROVED** [review]

**Pull request merged**